Modules/TRD/src/TrackletsTask.cxx

```diff
@@ -119,9 +119,9 @@ void TrackletsTask::retrieveCCDBSettings()
   if (mChamberStatus == nullptr) {
     ILOG(Info, Support) << "mChamberStatus is null, no chamber status to display" << ENDM;
   }
-  //j  else{
-  //   drawHashedOnHistsPerLayer();
-  //  }
+  // j  else{
+  //    drawHashedOnHistsPerLayer();
+  //   }
```

Comment:
Why was this modified? Or why is this still in the code, if it is commented out?

Reply:
Ok @tdietel ....I will remove all these...Thanks